### PR TITLE
Change to_dict and from_dict methods using Point arrays to use a to_array and from_array method on the Point classes #77

### PR DESCRIPTION
**Pull request by santiagogaray**
Hi @chriswmackey,

Still not confident with GitKraken,  I ended up copying the files with the changes to desktop, deleting and re-forking the repo, and overwriting it with the changed files.
What I don't like is that the diffs are entire files as opposed to the changes only.  Is this because of the override? 
Hope this still works, let me know!

### CODE REVIEW

**Comment by chriswmackey** [comment]
Thank you for sending this @santiagogaray . The actual changes that you made are good but we are definitely going to have to clean up the diffs before we merge this in or it will mess up a lot of the versioning history about which parts of the code were changed when. If you don't mind me going in to change your fork, I can see if I can fix it.

**Comment by chriswmackey** [comment]
I hope you don't mind that I just went in and fixed your fork in gitkraken. You can see the diffs are clean now.

File: ladybug_geometry/geometry3d/polyface.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def from_faces(cls, faces, tolerance=0):
         face_indices = []  # collection of face indices
         for f in faces:
             ind = []
-            loops = (f.boundary,) if not f.has_holes else (f.boundary,) + f.holes
+            loops = [f.boundary] if not f.has_holes else [f.boundary] + f.holes
```
Comment (chriswmackey) [inline ladybug_geometry/geometry3d/polyface.py]: I don't know how this change ended up in your fork but you are reverting an important bug fix that I committed before you started work on the sphere:
https://github.com/ladybug-tools/ladybug-geometry/commit/b11bfb127f1f74b491256babcdb30e30101d29f3

Please change it back to how it was (with tuples) and then we should be good to go.

**Comment by santiagogaray** [comment]
Thanks for cleaning up my mess @chriswmackey. Sorry for the pain, one day I will understand GitKraken better. All that said, I'm still not sure If what I just did is correct or not.

So, initially I noticed in my graph that we both had the same -to_array from_array- commit, in my fork and in the master.  Because of conflicts that this created would not allow me to merge my new commit. 
So I decided to revert my  -to_array from_array- commit and that seems like it solved the problem, hopefully. Is this correct?

Sorry again for the extra work for such as simple PR.

**Comment by chriswmackey** [comment]
No worries, @santiagogaray . Thanks for fixing the last change and this is all clear to merge in. There are ways to get rid of the extra revert and merging (without having it show up in the commit log) but it takes a little comfort with the "destructive" options in gitKraken and I know it's a little scary to use them at first.  In any case, squashing everything into one commit right now on the merge will make it all clean.

**Comment by santiagogaray** [comment]
Thanks @chriswmackey for cleaning this up. I will create a PR from the cone next.

**Pull request merged**